data/_nav.yml

```diff
@@ -179,15 +184,16 @@ reference:
         - url: /docs/reference/map-operations.html
           title: Map Specific Operations
 
-    - title: Multiplatform Programming
-      description: Reusing code across JVM, JS, and Native
-      content:
-        - url: /docs/reference/platform-specific-declarations.html
-          title: Platform-Specific Declarations
-        - url: /docs/reference/building-mpp-with-gradle.html
-          title: Building with Gradle
+    - title: Coroutines
+      description: Kotlin Coroutines Library
+      external:
+        path: kotlinx.coroutines/docs
+        nav: _nav.yml
+        base: /docs/reference/coroutines
+        repo: https://github.com/kotlin/kotlinx.coroutines
+        branch: master
 
-    - title: Other
+    - title: Various Language Features
```

Comment:
Looks like we have a problem here: this group of items has no meaningful name and the user will never guess what went under "Various language features". Please think about possible solutions to this, I'm happy to discuss options if needed